src/main/java/com/demoqa/shop/pages/AccountPage.java

```diff
@@ -18,18 +20,24 @@
     public WebElement button_register;
     @FindBy(how = How.XPATH, using = "//p[@class='login message']")
     public WebElement message_login;
+    @FindBy(how = How.XPATH, using = "//input[@id='username']")
+    public WebElement field_username_login;
+    @FindBy(how = How.XPATH, using = "//input[@id='password']")
+    public WebElement field_password_login;
```

Comment:
In my opinion all webelements from the page should be private, and have each corresponding public method, that will access correctly this webelement(buttons- click, textfields-sendkeys, and not vice versa by error)

Reply:
all webelements from the page are set as private and corresponding public method